.github/workflows/publish-draft-release.yml

```diff
@@ -6,6 +6,51 @@ on:
       - v**.**.**
 
 jobs:
+  build-runtimes:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        runtime: ['polkadot', 'kusama']
+    container:
+      image: chevdor/srtool:nightly-2020-07-20
```

Comment:
afaiu according to our [Docker images maintenance policy](https://github.com/paritytech/devops/wiki/Docker-images-maintenance-policy) only official images or images based on them should be used.